middleware/verify_jwt.js

```diff
@@ -0,0 +1,35 @@
+import jwt from 'jsonwebtoken';
+import { cartDao } from '../model';
+const TOKEN_KEY = '' + process.env.SECRET_KEY;
+
+const validateToken = async (req, res, next) => {
+  try {
+    let bearer, token;
+    const bearerHeader = req.headers.authorization;
+
+    if (bearerHeader) {
+      [bearer, token] = req.headers.authorization.split(' ');
+    } else {
+      const err = new Error('TOKEN_REQUIRED');
```

Comment:
```suggestion

    if (bearerHeader) {
      [_, token] = req.headers.authorization.split(' ');
    } else {
      const err = new Error('TOKEN_REQUIRED');
```
`bearer`이라는 변수는 따로 사용되지 않고 있네요, 사용되지 않는 변수인데도 일단 할당 자체는 필요할 때,
이렇게 언더바를 사용할 수 있습니다 👍

Reply:
네 감사합니다 !! 이렇게 한 번 수정해보도록 하겠습니다.

Reply:
@soheon-lee 소헌님, 그런데 이런 식으로 수정을 해주니 `ReferenceError: _ is not defined` 이러한 에러가 계속해서 출력되고 있는 상황입니다. 테스트 진행 시 header에 bearer '토큰' 이런 식으로 담아준 뒤 사용 중에 있는데, 그 때문에 bearer은 꼭 필요한 것이 아닐까 생각 중에 있습니다. 혹시 어떻게 생각하실까요 ?

Reply:
@hxyxneee 엇 그런가요? 진행될것이라고 생각하는데, 음 일단 알겠습니다 :) 원래대로 진행하시고, 이 관련 이슈는 나중에 리팩토링 한 번 해봐용